core/__tests__/models/app/modelCache.ts

```diff
@@ -0,0 +1,141 @@
+import { helper } from "@grouparoo/spec-helper";
+import { App } from "../../../src";
+import { AppsCache } from "../../../src/modules/caches/appsCache";
+
+describe("models/appsCache", () => {
+  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+
+  beforeEach(() => (AppsCache.TTL = 30 * 1000));
+  afterAll(() => (AppsCache.expires = 0));
+
+  beforeAll(async () => {
+    await helper.factories.properties();
+
+    const deletedApp = await helper.factories.app();
+    await deletedApp.update({ state: "deleted" });
+  });
+
+  describe("#findAllWithCache", () => {
+    test("it returns all the instances", async () => {
+      const instances = await AppsCache.findAllWithCache();
+      expect(instances.length).toBe(2);
+    });
+
+    test("it can filter to state", async () => {
+      const draftApp = await helper.factories.app();
+      await draftApp.update({ state: "draft" }, { hooks: false });
+
+      const instances = await AppsCache.findAllWithCache(undefined, "draft");
+
+      expect(instances.length).toBe(1);
+      expect(instances[0].id).toBe(draftApp.id);
+
+      await draftApp.destroy();
+    });
+
+    describe("rpc", () => {
+      let app: App;
+
+      async function makeApp() {
+        app = await helper.factories.app();
+        await helper.sleep(100); // wait for delayed RPC calls
+      }
+
+      afterEach(async () => {
+        try {
+          await app.destroy();
+        } catch {}
```

Comment:
```suggestion
      await app.destroy().catch(() => null);
```
You can do this too!